### PR TITLE
static pages, like authentication, wishlist are added

### PR DESCRIPTION
**Pull request by ShreyashMahajan**
In this PR, a static version of all the pages is added. A static version of the following pages is included.
1. Login.
2.Forgot password.
3. Sign up>
4. wishlist
5. cart page

please review the code and suggest for any improvement

### CODE REVIEW

File: src/pages/authentication/forgotPass/forgotPass.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
+import { Navbar } from "../../../components/navbar/navbar"
+import '../forgotPass/forgotPass.css'
```
Comment (Bhakti199) [inline src/pages/authentication/forgotPass/forgotPass.jsx:2]: export these components from single index file for easy access

File: src/pages/authentication/login/login.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
+import { Navbar } from "../../../components/navbar/navbar"
+import '../signUp/signUp.css'
+import '../login/login.css'
+import { Link } from "react-router-dom"
+
```
Comment (Bhakti199) [inline src/pages/authentication/login/login.jsx:5]: same here

**Review by Bhakti199 — APPROVED** [review]
overall code looks clean and well structured. This PR is good to merge.

**Review by JantuDeb — APPROVED** [review]

File: src/pages/authentication/forgotPass/forgotPass.jsx (hunk 1 above)
Comment (ShreyashMahajan) [reply src/pages/authentication/forgotPass/forgotPass.jsx:2]: ok, i will do that

**Pull request merged**